.sql/data.sql

```diff
@@ -0,0 +1,246 @@
+-- phpMyAdmin SQL Dump
+-- version 4.6.6deb1+deb.cihar.com~xenial.2
+-- https://www.phpmyadmin.net/
+--
+-- Host: localhost
+-- Czas generowania: 24 Paź 2018, 00:34
+-- Wersja serwera: 5.7.20-0ubuntu0.16.04.1
+-- Wersja PHP: 7.2.11-2+ubuntu16.04.1+deb.sury.org+1
+
+SET SQL_MODE = "NO_AUTO_VALUE_ON_ZERO";
+SET time_zone = "+00:00";
+
+--
+-- Baza danych: `adfind`
+--
+
+-- --------------------------------------------------------
+
+--
+-- Struktura tabeli dla tabeli `Category`
+--
+
+CREATE TABLE `Category` (
+  `id` int(11) NOT NULL,
+  `name` varchar(255) NOT NULL
+) ENGINE=InnoDB DEFAULT CHARSET=utf8;
```

Comment:
Poczytaj o tym: https://stackoverflow.com/questions/766809/whats-the-difference-between-utf8-general-ci-and-utf8-unicode-ci